src/css/default.css

```diff
@@ -62,7 +65,6 @@ body {
   background-color: var(--bgColor);
   color: var(--textColor);
   margin: 0;
-  transition: color 0.2s ease-out, background 0.2s ease-out;
```

Comment:
This transition is a core part of the problem

Reply:
That's what I though, huge thanks! 🙌🏻